### ISSUE TITLE
텐서 합치기랑 업샘플링

### ISSUE THREAD

**Issue by eddie94** [im-hjk/mask-offf-doodle#1]
텐서 두개를 합칠 때에는 torch.cat 함수를 이용하면 됩니다
forward 함수에 업샘플링 부분이 없으니 추가해 주시고
지금은 forward 함수에 리턴값이 없어서 아마 출력이 None이 나올겁니다
네트워크의 최종 출력 텐서를 반환해 주세요

**Comment by im-hjk** [comment]
알겠습니다
